tools/c7n_salactus/c7n_salactus/cli.py

```diff
@@ -33,7 +33,6 @@
 from rq.worker import Worker
 import tabulate
 
-from c7n.config import Bag
```

Comment:
this looks like a regression.. ie. the current location of Bag is c7n/config.py not c7n/utils.py

Reply:
Looks like my regression went too far back; set it to the current master value.